### PR TITLE
Angular 10

### PR DESCRIPTION
**Pull request by Bykiev**
Updated project to Angular 10, updated packages

### CODE REVIEW

**Comment by unocelli** [comment]
Hi Andrey, Thanks for the PR, Good job :rocket: I will do some test before merge it

File: client/package.json [hunk 1]
```diff
@@ -42,51 +42,49 @@
...
+    "panzoom": "^9.4.3",
+    "pdfmake": "^0.2.6",
     "rxjs": "~6.5.4",
     "socket.io-client": "^2.4.0",
-    "tslib": "^1.10.0",
+    "tslib": "^2.0.0",
     "uplot": "1.6.15",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
```
Comment (unocelli) [inline client/package.json:58]: Hi, maybe you know what I'm missing?

...\FUXA\client> ng lint --fix

Linting "FUXA"...
An unhandled exception occurred: Failed to load plugin '@angular-eslint' declared in '.eslintrc.json#overrides[0]': Class extends value undefined is not a constructor or nullReferenced from: C:\work\FUXA\pull-request\FUXA-PR-573\FUXA\client\.eslintrc.json      
See "C:\Users\defau\AppData\Local\Temp\ng-b8op7o\angular-errors.log" for further details.

Comment (Bykiev) [reply client/package.json:58]: Hi, sorry, missed to check it, I'll research the issue tomorrow. I think it's incompatible with eslint 8

Comment (Bykiev) [reply client/package.json:58]: Fixed by downgrading `eslint` package version

**Pull request merged**